profile/arc_wr_ro_crate.md

```diff
@@ -1,132 +1,103 @@
-
-```mermaid
-flowchart TD
-  classDef red fill:#f96,stroke:#333,stroke-width:2px;
-
-  subgraph Workflows
-    direction TB
-    A["Workflow(ComputationalWorkflow, LabProtocol)"]:::red
-    C["Subworkflow/Process A"]
-    D["Subworkflow/Process B"]
-    E["Subworkflow/Process AA"]
-    F["Subworkflow/Process AB"]
-    G["Subworkflow/Process AC"]
-  end
-
-  subgraph Runs
-    direction TB
-    B["Run(CreateAction, LabProcess)"]:::red
-    H["Run A"]
-    I["Run B"]
-    J["Run AA"]
-    K["Run AB"]
-    L["Run AC"]
-  end
-
-  A -- "hasPart" --> C
-  A -- "hasPart" --> D
-  C -- "hasPart" --> E
-  C -- "hasPart" --> F
-  C -- "hasPart" --> G
-
-  B -- "instrument" --> A
-  H -- "instrument" --> C
-  I -- "instrument" --> D
-  J -- "instrument" --> E
-  K -- "instrument" --> F
-  L -- "instrument" --> G
+runProcess --instrument--> workflowProtocol
+runProcess --executesLabProtocol--> workflowProtocol
+workflow --mainEntity--> workflowProtocol
 ```
 
-## Profile Requirements
+## Requirements
+
+### ARC Workflow
 
-This section describes the specifications for other Profiles that are used in the CWL RO-Crate Profile and deviate from their original 
-specifications.
+| Property | Required | Cardinality | Expected Type | Description |
+|----------|----------|-------------|---------------|-------------|
+| <h4>Required Properties</h4> | | | | |
+| **`@id`** | Required | ONE | [schema.org/Text](https://schema.org/Text)<br>OR [schema.org/URL](https://schema.org/URL) | Used to distinguish the resource being described in JSON-LD. |
+| **`@type`** | Required | ONE | [schema.org/Dataset](https://schema.org/Dataset) | Schema.org/Bioschemas class(es) for the resource declared using JSON-LD syntax. |
+| **`additionalType`** | Required | ONE | [schema.org/Text](https://schema.org/Text)<br>OR [schema.org/URL](https://schema.org/URL) | MUST be 'ARC Workflow' or ontology term to identify it as an ARC Workflow |
+| **`identifier`** | Required | ONE | [schema.org/Text](https://schema.org/Text)<br>OR [schema.org/URL](https://schema.org/URL) | A mandatory unique identifier, either a temporary identifier supplied by users or one generated by a repository or other database. For example, it could be an identifier complying with the LSID specification |
+| **`mainEntity`** | Required | MANY | [WorkflowProtocol](#workflow-protocol) | The Main Workflow Protocol described by the ARC Workflow. MUST follow the [Workflow Protocol profile](#workflow-protocol). |
```

Comment:
I think there should only be ONE main workflow? i.e. the `workflow.cwl` which in turn can point to other workflows.

Reply:
good catch